### ISSUE TITLE
aws_organizations_organization feature_set change should not force new resource

### ISSUE THREAD

**Issue by ryandeivert** [hashicorp/terraform-provider-aws#15462]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

Terraform version: 0.13.1
AWS Provider version: 3.9.0

### Affected Resource(s)

* `aws_organizations_organization`

### Terraform Configuration Files

When updating the following resource from the "before" to "after", terraform  the following `org` resource from:

#### Before
```hcl
resource "aws_organizations_organization" "org" {
  feature_set = "CONSOLIDATED_BILLING"
}
```

#### After
```hcl
resource "aws_organizations_organization" "org" {
  aws_service_access_principals = [
    "cloudtrail.amazonaws.com",
  ]

  feature_set = "ALL"
}
```

### Debug Output

```
-/+ aws_organizations_organization.org (new resource required)
      id:                                       "o-example" => <computed> (forces new resource)
      accounts.#:                               "2" => <computed>
      arn:                                      "arn:aws:organizations::123456789012:organization/o-example" => <computed>
      aws_service_access_principals.#:          "0" => "1"
      aws_service_access_principals.123456789: "" => "cloudtrail.amazonaws.com"
      feature_set:                              "CONSOLIDATED_BILLING" => "ALL" (forces new resource)
      master_account_arn:                       "arn:aws:organizations::123456789012:account/o-example/123456789012" => <computed>
      master_account_email:                     "email@foobar.com" => <computed>
      master_account_id:                        "123456789012" => <computed>
      non_master_accounts.#:                    "1" => <computed>
      roots.#:                                  "1" => <computed>
```

### Expected Behavior

The change to [`feature_set`](https://github.com/terraform-providers/terraform-provider-aws/blob/f8ba9a2c291dc809b220033c8050f2cea8d9a016/aws/resource_aws_organizations_organization.go#L157) does not force a new resource, which would generate a new Org ID. The feature set can be changed in Console without creating a new org ID, and I was informed by AWS that this should not need to generate a new org ID.

### Actual Behavior

A new resource if forced when `feature_set` is changed from `CONSOLIDATED_BILLING` to `ALL`. This should only force a new resource if going from `ALL` to `CONSOLIDATED_BILLING`, according to this note on this [page](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_support-all-features.html):

> The migration from consolidated billing features to all features is one-way. You can't switch an organization with all features enabled back to consolidated billing features only.

### Steps to Reproduce

1. `terraform apply` using the config above

### References

N/A

**Comment by philnichol** [comment]
Looks like it's possible to do this, although once you enable All Features it can't be reverted. I'd be interested in attempting to solve this if that's alright?

**Comment by ryandeivert** [comment]
@philnichol yep I believe that is the case. I just put a PR up for the fix here if you'd like to comment: https://github.com/terraform-providers/terraform-provider-aws/pull/15473

**Comment by philnichol** [comment]
@ryandeivert your solution is 1000x more elegant than what I was thinking :) didn’t even know about ForceNewIfChange, nice one!

**Comment by anGie44** [comment]
The proposed fix has been merged and will release with version `3.16.0` of the Terraform AWS Provider, expected out next Thursday.
